pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -202,8 +202,8 @@ func (q *Queue) Reconcile(ctx context.Context) (reconcile.Result, error) {
 		failedLaunches := lo.Filter(cmd.Replacements, func(r Replacement, _ int) bool {
 			return !r.Initialized
 		})
-		disruptionQueueFailedCounter.With(map[string]string{
-			methodLabel:            cmd.method,
+		disruptionQueueFailuresTotal.With(map[string]string{
+			methodLabel:            cmd.reason,
```

Comment:
Should this still be method? Should this not just match our decision metric where we have a reason and a decision?